src/psyclone/tests/test_files/gocean1p0/reordered_dependent_invoke.f90

```diff
@@ -0,0 +1,52 @@
+!-------------------------------------------------------------------------------
+! (c) The copyright relating to this work is owned jointly by the Crown,
+! Met Office and NERC 2014.
+! However, it has been created with the help of the GungHo Consortium,
+! whose members are identified at https://puma.nerc.ac.uk/trac/GungHo/wiki
+!-------------------------------------------------------------------------------
+! Author A. Porter STFC Daresbury Lab
+! Modified A. B. G. Chalk STFC Daresbury Lab
+
+PROGRAM dependent_invoke_test
+
+  ! Fake Fortran program for testing aspects of
```

Comment:
Please extend this to say what aspect this file is testing. (Also, are you sure there isn't one that already does this?)

Reply:
Removed